mlprimitives/primitives/numpy.ravel.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "numpy.ravel",
+    "contributors": [
+        "Alexander Geiger <geigera@mit.edu>"
+    ],
+    "documentation": "https://docs.scipy.org/doc/numpy/reference/generated/numpy.ravel.html",
+    "description": "Return a contiguous flattened array.",
+    "classifiers": {
+        "type": "helper"
+    },
+    "modalities": [],
+    "primitive": "numpy.ravel",
+    "produce": {
+        "args": [
+            {
+                "name": "y",
```

Comment:
I think it would be better to call the input `X` instead of `y`, as the input is expected to be a matrix, which `y` normally isn't. Also, manipulating `X` is much more frequent than `y`, so using `X` will probably mean having to rename less variables using `input_names` in the long run.